packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -13,11 +13,12 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.json']
```

Comment:
the order here needs to have `.mjs` first.

Reply:
@ljharb is this so if there's a conflict, it picks `.mjs`?

Reply:
Yes, exactly; that's how it will work in node as well.